doc/apimanager-reference.md

```diff
@@ -358,6 +365,12 @@ an `APIManager` status field should never be modified by the user.
 | --- | --- | --- | --- |
 | No fields for the moment | | | |
 
+## PersistentVolumeClaimResourcesSpec
+
+| **json/yaml field**| **Type** | **Required** | **Default value** | **Description** |
+| --- | --- | --- | --- | --- |
+| `requests` | [v1 Quantity](https://v1-17.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#quantity-resource-core) | Yes | N/A | Size of the PersistentVolumeClaim where the backup is to be performed. Set enough size to contain all [data that is backed up](#data-that-is-backed-up).
```

Comment:
The description here should be slightly changed as it does not deal with the backup in this case